routes/topics.js

```diff
@@ -5,12 +5,15 @@ const router = express.Router();
 // controller
 const topicsController = require('../controllers/topics');
 const {isAuth} = require('../middlewares/auth');
+const {validate} = require('../middlewares/joi-validation/validate');
+const {TopicValidation} = require('../joi-validations/topics.validations');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `validate` middleware and `TopicValidation` are imported but the old `QueryParamsValidationMiddleware` is not removed. If it's no longer used, consider removing it to avoid confusion.
```diff
- const QueryParamsValidationMiddleware = require('../middlewares/query-params-validation');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->